ts/store/actions/messages.ts

```diff
@@ -31,11 +34,27 @@ export type MessagesLoadFailure = Readonly<{
   error: true;
 }>;
 
+export type MessageLoadRequest = Readonly<{
+  type: typeof MESSAGE_LOAD_REQUEST;
+  payload: string;
+}>;
+
 export type MessageLoadSuccess = Readonly<{
   type: typeof MESSAGE_LOAD_SUCCESS;
   payload: MessageWithContentPO;
 }>;
 
+export type MessageLoadFailure = Readonly<{
+  type: typeof MESSAGE_LOAD_FAILURE;
+  payload: Error;
+  error: true;
+}>;
+
+export type MessageDetailsNavigate = Readonly<{
```

Comment:
keep the action type consistent with the action name: `NavigateToMessageDetails`

Reply:
Fixed.